.storybook/guides/deprecation.mdx

```diff
@@ -0,0 +1,45 @@
+import { Meta } from '@storybook/blocks';
+
+<Meta title="Guides/Deprecation workflow"/>
+
+# Deprecating a component
+
+When a component is deprecated, it is no longer recommended for use. Though the component will be removed from the codebase and will no longer receive feature updates, we will apply bug fixes to the package as necessary. We will also provide a migration path to a recommended alternative.
+
+## Deprecation process
+
+**Deprecation of a component is typically initiated and communicated by the design team.** They will indicate that a component is going to be deprecated and identify reccomended alternatives and proposed language around migrating away from that component's use.
+
+### Developer responsibilities
+
+#### Due diligence
+
+Before starting a deprecation, connect with the Spectrum Web Components team to align on the timing of the deprecation. Make sure downstream users are aware that the component is being deprecated and have a migration path to the recommended alternative. Give downstream consumers time to express any concerns or blockers.
+
+#### Storybook
+
+- [ ] Create a new folder with the component name in `.storybook/deprecated`, i.e., `.storybook/deprecated/quickaction`.
+- [ ] Copy `*.stories.js` and `metadata/*.yml` into the new folder (directly, not nested inside subfolders).
+- [ ] Open up and edit the title of the `*.stories.js` default export to be prefixed with the `Deprecated` category, i.e., `title: "Deprecated/Quick actions"`.
+- [ ] Update any local references to point to the package name instead, i.e.,<br/>_Original_:<br/>`import { Template } from "./template";`<br/><br/>_Updated to_:<br/>`import { Template } from "@spectrum-css/quickaction/stories/template.js";`.
+- [ ] In the parameters section of the `*.stories.js`, add `chromatic: { disable: true },` to ensure it no longer runs regression tests.
+
+#### Documentation
+
+- [ ] Update the status of the component to `Deprecated` in the `*.yml` file. Add any additional migration notes to the `deprecationNotice` keyword. i.e.,
+    ```yaml
+    name: Quick actions
+    status: Deprecated
+    deprecationNotice: Use an <a href="actionbar.html">action bar</a> to allow users to perform actions on either a single or multiple items at the same time, instead.
+    ```
+
+#### Codebase
+
+- [ ] Delete the package from the components directory.
+- [ ] Add the deprecated component's package and last version to the root package.json. i.e., `"@spectrum-css/quickaction": "^3.1.1",`.
```

Comment:
Pointing to the storybook package.json might be more clear than saying root here:

```suggestion
- [ ] Add the deprecated component's package and last version to `.storybook/package.json`. i.e., `"@spectrum-css/quickaction": "^3.1.1",`.
```

Reply:
Ah so this isn't a storybook package update, it's the `spectrum-css/package.json` - at the monorepo root.  What's a better way for me to communicate that?

Reply:
Ohh I think I'm confused since we didn't do that in quickactions, it was just a [storybook package change](https://github.com/adobe/spectrum-css/pull/2304/files#diff-ff15ba6afb0028b97d1f935edfc11920354ec64a4e3b2db849f7f03a27eb7735). Should that have happened at the root instead?

Reply:
I added it later - we realized it was breaking the docs reference for card :)

Reply:
Gotcha! Thanks for the info